index.html

```diff
@@ -24,8 +24,127 @@ <h1>I’m Mariane Glad to see you!</h1>
             <li class="social" ><a href=""><img src="assets/Vector2.png" alt=""></a></li>
             <li class="social" ><a href=""><img src="assets/Vector3.png" alt=""></a></li>
         </ul>
-        
     </div>
     </section>
+    <section class="works">
+        <div class="cards">
+            <img src="./assets/Snapshoot_Portfolio.png" alt="">
+            <div class="card1">
+            <h2>Tonic</h2>
+            <ul> 
+                <li>CANOPY</li>
+                <li>⬤</li>
+                <li>BackEnd Dev</li>
+                <li>⬤</li>
+                <li>2015</li>
+            </ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="languages">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li>
+            </ul>
+            <button>See Project</button>
+        </div>
```

Comment:
- [ ] Please implement the button interactions for all the buttons in the works and about section(enable, hover, etc.).

![image](https://user-images.githubusercontent.com/83803414/185662402-e9e1c6fe-0c3b-4b7c-93a1-b5c3134bb99d.png)

Reply:
how can i find the button interaction in figma?

Reply:
@thecodechaser

Reply:
@shakerAbuDrais Hi, scroll to the right, It's in the middle between desktop and mobile version